<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Reservar</class>
 <widget class="QWidget" name="Reservar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>20</y>
     <width>131</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HP Simplified Light</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>RESERVAR</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtCodR">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>140</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HP Simplified Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtCodTT">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>60</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HP Simplified Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>270</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HP Simplified Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ingrese el código del tren: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>270</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HP Simplified Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ingrese el código de la ruta: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>270</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HP Simplified Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ingrese el código del tipo de tren: </string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtCodT">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>100</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HP Simplified Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtCodNA">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>220</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HP Simplified Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtCodNP">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>180</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HP Simplified Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>230</y>
     <width>270</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HP Simplified Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ingrese el número de asientos: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>331</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HP Simplified Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ingrese el número de personas que viajan: </string>
   </property>
  </widget>
  <widget class="QPushButton" name="btSalir">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>420</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HP Simplified Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btAP">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>380</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HP Simplified Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>AGREGAR PASAJEROS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btReservar">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>420</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HP Simplified Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Reservar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>270</y>
     <width>371</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HP Simplified Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ingrese el número de pasaporte del comprador: </string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtCodPas">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>260</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HP Simplified Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
